SumoLogic.Logging.Serilog/SumoLogic.Logging.Serilog.csproj

```diff
@@ -26,7 +26,7 @@
         <ProjectReference Include="..\SumoLogic.Logging.Common\SumoLogic.Logging.Common.csproj" />
     </ItemGroup>
 	<ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-            <PackageReference Include="Serilog" Version="[2.0,4.0)" />
+            <PackageReference Include="Serilog" Version="[2.0,)" />

```

Comment:
I don't have enough context here, if this is bumping up to allow Serilog 4.*, why is the change removing 4.0?

Reply:
Ah gotcha, didn't noticed the bracket and parenthesis. Makes alot more sense, approved!